alcs-frontend/src/app/features/board/board.component.ts

```diff
@@ -105,6 +106,60 @@ export class BoardComponent implements OnInit, OnDestroy {
     this.destroy.complete();
   }
 
+  async onSelected(card: CardSelectedEvent) {
+    this.setUrl(card.uuid, card.cardType);
+    switch (card.cardType) {
+      case CardType.APP:
+        await this.openAppCardDetailDialog(card.uuid);
+        break;
+      case CardType.RECON:
+        await this.openReconCardDetailDialog(card.uuid);
+        break;
+      case CardType.PLAN:
+        await this.openPlanningCardDialog(card.uuid);
+        break;
+      default:
+        console.error('Card type is not configured for a dialog');
```

Comment:
Should we show an error toast to user "Failed to open card". I know that this should not happen but if we already providing a error message here

Reply:
Done!